iwyu_test_util.py

```diff
@@ -45,6 +45,26 @@
 # // IWYU_ARGS: -Xiwyu --mapping_file=... -I .
 _IWYU_TEST_RUN_ARGS_RE = re.compile(r'^//\sIWYU_ARGS:\s(.*)$')
 
+# This is a line that specifies when a test should be skipped (because
+# the result would not be valid). IWYU_REQUIRES specifies a condition
+# that must be true for the test to execute. IWYU_UNSUPPORTED
+# specifies a condition in which the test is invalid, and would be
+# skipped. Multiple IWYU_REQUIRES and IWYU_UNSUPPORTED can be
+# specified, but they must be in a single comment block. For a test to
```

Comment:
> but they must be in a single comment block

This seems like an artificial constraint. Isn't parsing much more straightforward if you skip this rule?